lib/routes/notifications-event.js

```diff
@@ -0,0 +1,19 @@
+const router = require('koa-router')()
+const models = require('../models')
+
+router
+  .get('/notifications/event/:eventId', async (ctx) => {
+    let eventId = ctx.params.eventId
+    ctx.body = await models.db.notification.findAll({
+      where: {
+        event_id: eventId
+      },
+      order: [['priority', 'ASC'], ['message_date', 'DESC']],
```

Comment:
Switch the order of these, date should be the first sorting mechanism and then priority

Reply:
Do we want expired messages to be excluded from this call or will that logic be done on the client?

Reply:
Notifications by event_id will be used admin console, which can handle its own filtering if needed. Client Apps will not use this API.

Reply:
Revised